embeddings/transformation/hf_transformation/sampling_transformation.py

```diff
@@ -1,16 +1,17 @@
+import abc
 from typing import Optional
 
 import datasets
+import sklearn.model_selection
+from datasets import DatasetDict
 
 from embeddings.transformation.transformation import Transformation
 from embeddings.utils.loggers import get_logger
 
 _logger = get_logger(__name__)
 
 
-class SampleSplitsHuggingFaceTransformation(
-    Transformation[datasets.DatasetDict, datasets.DatasetDict]
-):
+class SampleSplitTransformation(Transformation[datasets.DatasetDict, datasets.DatasetDict]):
```

Comment:
```suggestion
class SampleSplitsTransformation(Transformation[datasets.DatasetDict, datasets.DatasetDict]):
```
for the coherence with other names